diff --git a/gatsby-config.js b/gatsby-config.js
index e9f3c8c..2575838 100644
--- a/gatsby-config.js
+++ b/gatsby-config.js
@@ -213,6 +213,25 @@ module.exports = {
         // Raven.js version, this is optional.
         version: "3.19.1"
       }
+    },
+    {
+      resolve: "@halkeye/gatsby-source-goodreads",
+      options: {
+        developerKey: "4Bgp3HlSJnyq5e9YsExNQ",
+        goodReadsUserId: "3300733"
+      }
+    },
+    {
+      resolve: "gatsby-source-strava-activities",
+      options: {
+        // This is the "Access Token" from:
+        // https://www.strava.com/settings/api
+        authToken: "d02766eda6daf54ee78d41aac56588b863af5103",
+        // [Optional] An epoch timestamp to use for filtering activities that have taken place after a certain time.
+        after: null,
+        // An epoch timestamp to use for filtering activities that have taken place before a certain time.
+        before: null
+      }
     }
   ]
 };
